frontend/src/App.css

```diff
@@ -1,6 +1,3 @@
 #root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+ font-family: 'BC Sans',
```

Comment:
We could move this to be `App.style.tsx` like you did for the `Header.style.tsx` and keep it a bit more consistent. What do you think?

Reply:
I wouldn't touch that for now. For now, this ticket is only for header. Probably on Landing Page ticket we can implement that.